packages/terra-functional-testing/package.json

```diff
@@ -73,7 +73,8 @@
     "uuid": "^3.0.0",
     "webdriverio": "^6.5.0",
     "webpack-dev-server": "^3.11.0",
-    "which": "^2.0.2"
+    "which": "^2.0.2",
+    "strip-ansi": "^6.0.0"
```

Comment:
This should be listed alphabetically.

Reply:
https://github.com/cerner/terra-toolkit/pull/665/files#diff-72c2847b50d4152bd910ef25c33dafbd31a04d507329a333036e6226e5e8ef71R73